packages/hooks/src/test/index.test.js

```diff
@@ -943,3 +945,86 @@ test( 'checking hasFilter with named callbacks and removeAllActions', () => {
 	expect( hasFilter( 'test.filter', 'my_callback' ) ).toBe( false );
 	expect( hasFilter( 'test.filter', 'my_second_callback' ) ).toBe( false );
 } );
+
+describe( 'async filter', () => {
```

Comment:
Should we also add tests that verify the functions that work with the current hook work well with async filters/actions? `doingAction` comes to mind.

Reply:
Added tests that verify that `doingAction`/`doingFilter` return correct results when multiple hooks are running at once, in parallel.